src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -211,10 +211,11 @@ public bool RunPreviewIsChecked
         {
             get
             {
-                return dynamoViewModel.ShowRunPreview;
+                return preferenceSettings.ShowRunPreview;
```

Comment:
Does the orange node outline still show up in manual mode without setting this?

Reply:
The orange node outline is showing up for the newly modified nodes but when the workspace is reopened and the toggle button is switched on, it is not picking up the setting. Looking into it.

Reply:
@QilongTang Previously whenever the current workspace was closed, the RunType on DynamoViewModel was being reset to Automatic and that would reset the ShowRunPreview flag to 'false'. I have changed that behavior for ShowRunPreview, so that the ShowRunPreview flag will continue to honor the user preference.

Reply:
dynamoViewModel.ShowRunPreview is being set in the setter method whenever RunPreviewIsChecked control is changed. So the orange outline functionality would be fine.